core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -562,10 +562,10 @@ $(document).ready(function () {
 
                 $("#showAdditionalMetrics").append(
                     "<div><a id='additionalMetrics'>" +
-                    "<span class='expand-input-rate-arrow arrow-closed' id='arrowtoggle-optional-metrics'></span>" +
+                    "<span class='expand-input-rate-arrow arrow-closed' id='arrowtoggle-optional-metrics'></span> " +
                     "Show Additional Metrics" +
                     "</a></div>" +
-                    "<div class='container-fluid container-fluid-div' id='toggle-metrics' hidden>" +
+                    "<div class='container-fluid-div ml-4 d-none' id='toggle-metrics'>" +
```

Comment:
why do we need `ml-4 d-none`

Reply:
The ml-4 is needed to add the left margin on the set of checkboxes so that it is indented similarly to the original / current formatting.  Without the ml-4 class the checkboxes are flush with the "Show Additional Metrics" arrow.

With regards to d-none, jQuery 1.8 deprecated .toggle() and jQuery 1.9 removed it so I had to change the JS for the "Show Additional Metrics" expand/collapse functionality.  I decided to change it to the jQuery .toggleClass() method and use the Bootstrap provided d-none (display none) class to show and hide the div instead of creating a new custom class.

Reply:
I see. How about putting the margin-left into the CSS of `container-fluid-div` so that we don't need to write 'ml-4' every time.